### PR TITLE
Module 1: Adding a contact form to the mobile version

### PR DESCRIPTION
**Pull request by alvinlouis29**
#This is the Third milestone in creating my Personal Portfolio website, in this section I have created a form and added it to the contact section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -193,6 +193,71 @@ <h1>Skills</h1>
...
+                                <label for="name">Full Name</label>
+                                <input type="text" id="full_name" name="full_name">
+                              </li>
+                              <li>
+                                <label for="email">E-mail</label>
+                                <input type="email" id="email" name="user_email">
+                              </li>
+                              <li>
+                                <label for="msg">Message</label>
+                                <textarea id="message" name="user_message"></textarea>
```
Comment (dasileker) [inline index.html:239]: - [ ] The **textarea & inputs** should have a ** maxlength** and also should be required.

lflf

File: style.css [hunk 2]
```diff
@@ -47,13 +51,21 @@ a {
   justify-content: center;
 }
 
+form ul {
+  margin: 0;
+}
+
```
Comment (dasileker) [inline style.css:57]: - [ ] Your form is not well displayed for both views mobile and desktop

ppppp

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @alvinlouis29,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- Your Portfolio is not well  displayed in the mobile or the desktop views, so please try to make the require changes + fix the view. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (alvinlouis29) [reply style.css:57]: We are doing mobile view on this part of the project, no Desktop view yet. And the form is displaying well in my mobile view from 320px to 500px.

File: index.html (hunk 1 above)
Comment (alvinlouis29) [reply index.html:239]: Thank you very much let me work on it

**Review by dasileker — APPROVED** [review]
### **Hi @alvinlouis29,** 

### **Project Approved ✅**

Approved


Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 

### **Optional**

- Your contact form is good, but it still send an empty message and no one would be happy to receive an empty message, so please add **required** also to the **`<textarea>`** to fix that 👍.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**